content/cloud-computing/concepts/saas/saas.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'SaaS'
+Description: 'SaaS is a cloud computing service delivery model that makes software and applications available over the internet.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'back-end-engineer-career-path'
+---
+
+Software as a service or **SaaS** is a cloud computing service delivery model that makes software and applications available over the internet.
+
+Some examples of SaaS include Google Workspace, Dropbox, and Microsoft 365.
+
+## Architecture
+
+SaaS services typically use a multi-tenant cloud architecture. This allows consumers to share computing resources within a cloud.
+
+Each tenant has their own space to store data. The data is isolated and will remain invisible to other tenants.
+
+## Benefits
+
+- Accessible: Requires only an internet connection, can be accessed from any internet-enabled device and location.
+- Management: Users do not have to maintain anything; all maintenance and updates are provided by the service provider.
+- Scalability: Users can access as many services as they want.
```

Comment:
Would change Scalability to something like "Users can use as much of the service as they want, and are charged for the amount used.